docs/blog/2018-12-17-turning-the-static-dynamic/index.md

```diff
@@ -51,12 +51,14 @@ Let's walk through the steps:
     "start": "run-p start:**",
     "start:app": "npm run develop",
     "start:lambda": "netlify-lambda serve src/lambda",
-    "build": "run-p build:**",
+    "build": "gatsby build && netlify-lambda build src/lambda",
     "build:app": "gatsby build",
     "build:lambda": "netlify-lambda build src/lambda",
   },
 ```
 
+When deploying to Netlify, `gatsby build` must be run before `netlify-lambda build src/lambda` or else your Netlify function builds will fail.  To avoid this, do not set your build script command to `"build": "run-p build:**"` when you replace `scripts` in `package.json`.  Doing so will run all build scripts in parallel.  This will make it possible for `netlify-lambda build src/lambda` to run before `gatsby build`.  
```

Comment:
```suggestion
When deploying to Netlify, `gatsby build` must be run before `netlify-lambda build src/lambda` or else your Netlify function builds will fail. To avoid this, do not set your build script command to `"build": "run-p build:**"` when you replace `scripts` in `package.json`.  Doing so will run all build scripts in parallel. This will make it possible for `netlify-lambda build src/lambda` to run before `gatsby build`.  
```